caproto/threading/client.py

```diff
@@ -433,8 +433,8 @@ def send(self, port, *commands):
                 else:
                     self.udp_sock.sendto(bytes_to_send, (host, port))
             except OSError as ex:
-                raise OSError(f'{ex} while sending {len(bytes_to_send)} '
-                              f'bytes to {host}:{port}') from ex
+                raise CaprotoNetworkError(
+                    f'{ex} while sending {len(bytes_to_send)} bytes to {host}:{port}') from ex
```

Comment:
We should pick either `{host}:{port}` or `({host}, {port})` and stick with it. My preference is the former as it's standard notation.

Reply:
Yeah, fair. I think we agree that `(host, port)` is a useful internal representation so that `port` can be an `int`, but you are right that we shouldn't expose that out.